components/screens/menu/Version.tsx

```diff
@@ -0,0 +1,15 @@
+import * as Application from 'expo-application';
```

Comment:
nit: this file doesn't contain any JSX, so we usually just use a `.ts` extension for that.